docs/UserGuide.md

```diff
@@ -232,10 +279,13 @@ You can view all medication information by its name.
 
 Format: `view -n MEDICATION_NAME`
 
-Example: `view -n medication_b`
+Example: `meditracker> view -n medication_b`
```

Comment:
Similar to the above, can remove underscore for consistency

Reply:
On a separate note, is `meditracker>` needed?

Reply:
Removed the underscore in  aff146c

@annoy-o-mus Yes, to keep the "Example" format consistent throughout the UG